src/tuning/kernels/xgemm.cpp

```diff
@@ -54,7 +58,14 @@ class TuneXgemm {
   static size_t DefaultBatchCount() { return 1; } // N/A for this kernel
   static double DefaultFraction() { return (V==1) ? 1.0 : 512.0; } // test all or sample randomly
   static size_t DefaultNumRuns() { return 2; } // run every kernel this many times for averaging
-
+  static size_t DefaultNumSearchStragegy() { return 2; } // Full search and Random/PSO
+  static size_t DefaultSwarmSizePSO() { return 8; } 
+  static double DefaultInfluenceGlobalPSO(){ return 0.1; }
+  static double DefaultInfluenceLocalPSO(){ return 0.3; }
+  static double DefaultInfluenceRandomPSO(){ return 0.6; }
+  static size_t DefaultHeuristic(){ return size_t{3};} // PSO
```

Comment:
It's a pity this comment "// PSO" is needed (and similar ones elsewhere). It would be much nicer to have a `scoped enum` instead of encoding these heuristic-choices with integers directly. That would make the code clearer and will no longer require these comments :-)